src/main/java/com/jamieswhiteshirt/reachentityattributes/ReachEntityAttributes.java

```diff
@@ -44,6 +45,14 @@ public static double getSquaredAttackRange(final LivingEntity entity, final doub
         return attackRange * attackRange;
     }
 
+    public static boolean isInInventoryValidationRange(final PlayerEntity player, final double sqBetweenDistance, final double sqReachDistance) {
+        return sqBetweenDistance <= getSquaredReachDistance(player, sqReachDistance);
+    }
+
+    public static double getInventoryValidationValue(final PlayerEntity player, final double sqBetweenDistance, final double sqReachDistance) {
+        return ReachEntityAttributes.isInInventoryValidationRange(player, sqBetweenDistance, sqReachDistance) ? Double.NEGATIVE_INFINITY : Double.POSITIVE_INFINITY;
```

Comment:
Why does this use negative and positive infinity?

Reply:
Okay okay, I've immediately thought of a better way at going about this.

Reply:
Soo, the reasoning behind this is the return method of where it's redirecting.

`return playerEntity.squaredDistanceTo((double)blockPos.getX() + 0.5, (double)blockPos.getY() + 0.5, (double)blockPos.getZ() + 0.5) <= (double)(i * i);`

I need to guarantee this returning true or false, but I can really only redirect over the playerEntity.squaredDistanceTo.

If I were to ModifyVariable i, you'd miss out on specific values because you'd have to cast them to an int.